src/commands/log/mod.ts

```diff
@@ -149,6 +154,68 @@ function fmtSession(idx: string, sess: Session, indentWidth: number) {
     return strings.join("");
 }
 
+function printTimesheet(sessions: Session[]) {
+    // bucket sessions into days
+
+    let buckets: Map<number, number> = new Map();
+    let table: string[][] = [];
+    sessions.forEach((session) => {
+        let start = new Date(session.start);
+        let end = new Date(session.end ? session.end : Date.now());
+        const [startyear, startmonth, startdate] = destructureDate(start);
```

Comment:
I'd prefer if these variable names were shorter, maybe starty,startm,startd or similar.

Reply:
makes it more readable, so I want to keep this around

Reply:
In that case, please fix the casing, and we should be good to merge!